### ISSUE TITLE
表格中网络图片显示

### ISSUE THREAD

**Issue by HistoryZhang** [TooBug/wemark#47]
表格中加入网络图片标签后，![]()，图片在表格中无法显示。

**Comment by HistoryZhang** [comment]
现在图片是按照屏幕宽度来等比放大的么？我有个小图，然后解析出来好大，效果失真了；

**Comment by TooBug** [comment]
是的。考虑编辑图片，加留白。markdown本身不含图片格式的信息，一般就是100%宽度或者居中，目前wemark是统一做100%宽度处理。

**Comment by ZhuPeng** [comment]
@HistoryZhang  在表格内额外添加了 图片和链接的支持，参考：  https://github.com/ZhuPeng/mp-githubtrending/blob/master/third-party/wemark/wemark.wxml#L9

图片大小问题，如果不是太小的表格自动化分割，太小的图片需要在加载后设置大小，我这边实现了一个简单的版本：https://github.com/ZhuPeng/mp-githubtrending/blob/master/third-party/wemark/wemark.js#L41
